homeassistant/components/arcam_fmj/config_flow.py

```diff
@@ -0,0 +1,34 @@
+"""Config flow to configure the SimpliSafe component."""
+from collections import OrderedDict
+from operator import itemgetter
+import logging
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PORT
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+_GETKEY = itemgetter(CONF_HOST, CONF_PORT)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class ArcamFmjFlowHandler(config_entries.ConfigFlow):
+    """Handle a SimpliSafe config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Initialize the config flow."""
+        self._config = OrderedDict()
+
+    async def async_step_import(self, import_config):
```

Comment:
So why only the import? That seems a bit inconvenient for the user?

Reply:
Because without configuring a turn_on service call with an IR blaster or similar it's quite limited in use. And there is no good way to do that from config flows at the moment. Perhaps once option flows are in place...